lib/src/base-styles/_nx-colors.scss

```diff
@@ -50,11 +50,34 @@
   --nx-color-validation-invalid-icon: var(--nx-color-validation-invalid);
   --nx-color-validation-invalid-message: var(--nx-color-validation-invalid);
 
-  --nx-color-alert-info: var(--nx-swatch-blue-40);
   --nx-color-alert-success: var(--nx-swatch-green-25);
```

Comment:
I wonder if we should add a ticket to the end of the epic to further clean this up.  I don't think we should do it in this PR, since it affects other PRs that are in flight.  But I feel like the old names for these two are no longer ideal.  They are no longer especially associated with Alerts.  Alerts do use them in some cases, but so do other components.  I wonder if they should be renamed to e.g. `--nx-color-success`, with the existing names deprecated.

Reply:
> I wonder if we should add a ticket to the end of the epic to further clean this up. I don't think we should do it in this PR, since it affects other PRs that are in flight. But I feel like the old names for these two are no longer ideal. They are no longer especially associated with Alerts. Alerts do use them in some cases, but so do other components. I wonder if they should be renamed to e.g. `--nx-color-success`, with the existing names deprecated.

I agree, so ticket is made and in the backlog: https://issues.sonatype.org/browse/RSC-1566